### PR TITLE
Information about you - Help text update

### PR DESCRIPTION
**Pull request by Jay-Neighbours-I72**
Tweaking help text

### CODE REVIEW

File: src/config/locales/en/identification.js [hunk 1]
```diff
@@ -25,31 +25,31 @@ export const identification = {
...
           'If no middle name leave this field empty and check the "No middle name" checkbox.'
         ],
-        note: 'Note: "Initial only" is for single letter names only, not for the initial of your full name.'
+        note: 'Note: If this middle name is a single letter only type that letter and check the "Initial only" checkbox.'
       }
     },
     suffix: {
       help: {
-        title: 'Need help with this suffix?',
+        title: 'Use "Other" is your suffix isn\'t listed?',
```
Comment (bmallred) [inline src/config/locales/en/identification.js]: @Jay-Neighbours-I72 should this read

> Use "Other" **if** your suffix isn't listed?

_is_ -> _if_

File: src/config/locales/en/identification.js [hunk 2]
```diff
@@ -94,7 +93,7 @@ export const identification = {
   birthdate: {
     title: 'Provide your date of birth',
     help: {
-      title: 'Need help with your birthdate?',
+      title: 'Check "Estimated if date of birth is unknown?',
```
Comment (bmallred) [inline src/config/locales/en/identification.js]: Missing closing double quote.

File: src/config/locales/en/identification.js (hunk 1 above)
Comment (Jay-Neighbours-I72) [reply src/config/locales/en/identification.js]: + Good catch

File: src/config/locales/en/identification.js (hunk 2 above)
Comment (Jay-Neighbours-I72) [reply src/config/locales/en/identification.js]: Good catch

**Pull request merged**